packages/foreman/foreman-obsolete-packages/foreman-obsolete-packages.spec

```diff
@@ -0,0 +1,24 @@
+Name: foreman-obsolete-packages
+Version: 3.6
+Release: 1%{?foremandist}%{?dist}
+License: Public Domain
+Summary: A package to obsolete retired packages
+URL: https://github.com/theforeman/foreman-packaging
+
+Obsoletes: rubygem-fog-google < 1.19.0-2
```

Comment:
@evgeni do we also want to add some Foreman plugins we recently removed, even if there is no DB cleanup?

Reply:
I'd prefer not to *right now*.

Doing so would leave cruft in the DB and with the pkg gone it's hard to say where the cruft originated from.